examples/vae/vae.py

```diff
@@ -46,10 +46,10 @@ def forward(self, x):
 class Decoder(nn.Module):
     def __init__(self, z_dim, hidden_dim):
         super(Decoder, self).__init__()
-        # setup the three linear transformations used
+        # setup the two linear transformations used
         self.fc1 = nn.Linear(z_dim, hidden_dim)
         self.fc21 = nn.Linear(hidden_dim, 784)
-        # setup the non-linearity
+        # setup the non-linearities
```

Comment:
looks good but you have `fudge` in the decoder in vae.py but not in the notebook?

Reply:
Good point; it's not needed. Removed.

Reply:
but can you confirm you can get good numbers (no crashing etc.) without the fudge?

Reply:
The test graph was generated from the notebook itself, so I can confirm that it doesn't crash, and gives reasonable results. Let me generate some sample images too, and validate.